Source/WebKit/UIProcess/Automation/protocol/BidiScript.json

```
@@ -0,0 +1,174 @@
+{
+    "domain": "BidiScript",
+    "exposedAs": "script",
+    "description": "The script module contains commands and events relating to script realms and execution.",
+    "condition": "ENABLE(WEBDRIVER_BIDI)",
+    "types": [
+        {
+            "id": "EvaluateResultType",
+            "type": "string",
+            "enum": [ "success", "exception" ]
+        },
+        {
+            "id": "EvaluateResultOwnership",
```

Should we use `ResultOwnership` instead, to match the spec term? It will also be used serializing data that comes from the browser through the `script.Channel` type.

https://w3c.github.io/webdriver-bidi/#type-script-ResultOwnership

```suggestion
            "id": "ResultOwnership",
```